Assets/XML/GameText/Units_CIV4GameText.xml

```diff
@@ -4379,6 +4379,7 @@
 			<Plural>0:0:0:0:1:1:1:1</Plural>
 		</German>
 		<Russian>Ãðóïïà ëþäåé</Russian>
+    <Italian>Comitiva ancestrale</Italian>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_BANDEIRANTE_PEDIA</Tag>
```

Comment:
The only thing I have to nitpick on here. tabulation ^^

Reply:
For what I can see, there are also 4 blanks in the file Promotions_CIV4GameText.xml (sorry).

Reply:
![image](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/728dba7d-89dc-45d4-9dd7-534fd7b35c1d)

Reply:
![image](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/0057f184-6e84-4d91-a4d4-4ad501402c1b)

Reply:
Nothing else I hope, checked multiple times

Reply:
> Nothing else I hope, checked multiple times

Nothing else. ^^

Reply:
well then this time it has gone better, but there' s still room to improve... :)

Reply:
Hmm, the tabulation is still wrong.... what text editor are you using?

Reply:
Ok it should be good now and from now on I know I can correct them myself. Thanks and good night!